Libraries/Network/NetInfo.js

```diff
@@ -168,6 +168,9 @@ const _isConnectedSubscriptions = new Map();
  * ```
  */
 const NetInfo = {
+  /*
+   * Invokes the handler whenever the network info changes. The handler is passed with any of the connectivity types listed above.
```

Comment:
Can you change it to `Invokes the listener whenever network status changes. The listener receives one of the connectivity types listed above.`